src/FsAutoComplete/LspServers/AdaptiveFSharpLspServer.fs

```diff
@@ -827,9 +823,11 @@ type AdaptiveFSharpLspServer(workspaceLoader: IWorkspaceLoader, lspClient: FShar
           |> Array.distinctBy (fun e ->
             e.Severity, e.ErrorNumber, e.StartLine, e.StartColumn, e.EndLine, e.EndColumn, e.Message)
 
-        let uri = Path.LocalPathToUri(file.Lines.FileName)
-        let diags = errors |> Array.map fcsErrorToDiagnostic
-        diagnosticCollections.SetFor(uri, "F# Compiler", diags)
+        NotificationEvent.ParseError(errors, file.Lines.FileName)
+        |> notifications.Trigger
+
+
+        do! analyzeFile config (file.Lines.FileName, file.Version, file.Lines, parseAndCheck)
```

Comment:
Makes sure analyzers finish before we move on and create many threads doing analyzers